Microsoft.Toolkit.Uwp.UI.Controls/HamburgerMenu/HamburgerMenu.xaml

```diff
@@ -1,6 +1,12 @@
-﻿<ResourceDictionary xmlns="http://schemas.microsoft.com/winfx/2006/xaml/presentation"
-                    xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
-                    xmlns:local="using:Microsoft.Toolkit.Uwp.UI.Controls">
+﻿<ResourceDictionary 
```

Comment:
What do you mean "what unnecessary changes?" Look at the diff! If all you did to the file was add the template and visual state, that should be the only changes I see in that file. They aren't, you've changed the style of the xaml.